pkg_defs/annie/build.sh

```diff
@@ -1,2 +1,2 @@
 export SKA=/dev/null
```

Comment:
Can we instead just ensure that the `SKA` is set in `ska_builder.py` i.e. if currently undefined then set to `/proj/sot/ska`.  Currently `cheta` is strict and won't import without finding an archive, while `mica` will import as long as `SKA` is defined to something.  This is somewhat a philosophical discussion about whether a package should import if it does not have the data to actually function, and perhaps situations like this show the way (contrary to my past thoughts).

The right answer from the packaging perspective is moving to single-source version so `setup.py` can run without importing the package, but in the meantime I think we can clean up these recipes assuming integration with `ska_builder.py` (which is already the case with the `SKA_TOP_SRC_DIR`).

Reply:
I was a little concerned, since our default build will likely be on a Linux machine that will see the real /proj/sot/ska and run as the aca user, that giving any more recipes access to /proj/sot/ska increased risk.

Reply:
~~Fair enough to not change the ska_builder.py script right now, instead just use `script env SKA=whatever python setup.py install`.~~

Nope, that won't work on Windows.  So back to setting `SKA` to something in ska_builder.py.  Honestly I'm not that concerned about `python setup.py install` ever *writing* to something in SKA if it exists.  What happens there is importing various packages (primarily to get the version), and I always consider it safe to import any Ska package as `aca`.  If the act of importing a package can corrupt our flight database then we have bigger problems.

Did you have some specific failure path in mind that I'm missing?

Nevertheless we can be super careful just the same in ska_builder.py by making a temp dir, point SKA at that, and then make a dir SKA/eng_archive/data.  That will satisfy cheta and mica to get them to import.